main.js

```diff
@@ -1,5 +1,5 @@
 // Variables
-var newPalette = document.querySelector('#newPalette');
+var newPaletteBtn = document.querySelector('#newPaletteBtn');
 var allLocks = document.querySelectorAll('.lock');
```

Comment:
Awesome job consolidating all of our variables using querySelectorAll! This removed a significant number of code lines.